src/CMakeLists.txt

```diff
@@ -332,51 +329,60 @@ if (MSVC)
 	)
 endif()
 
-add_executable(${EXECUTABLE} ${SOURCE})
-
-
-# Output the executable into the $/Server folder, so that it has access to external resources:
-set(EXECUTABLE_OUTPUT_PATH ${CMAKE_SOURCE_DIR}/Server)
-SET_TARGET_PROPERTIES(${EXECUTABLE} PROPERTIES
-	RUNTIME_OUTPUT_DIRECTORY_DEBUG          ${CMAKE_SOURCE_DIR}/Server
-	RUNTIME_OUTPUT_DIRECTORY_RELEASE        ${CMAKE_SOURCE_DIR}/Server
-	RUNTIME_OUTPUT_DIRECTORY_DEBUGPROFILE   ${CMAKE_SOURCE_DIR}/Server
-	RUNTIME_OUTPUT_DIRECTORY_RELEASEPROFILE ${CMAKE_SOURCE_DIR}/Server
-)
+if (NOT ANDROID)
+	add_executable(${CMAKE_PROJECT_NAME} ${SOURCE})
```

Comment:
What @Schwertspize said.

Also, for the variable itself, I thought, since project_name = executable, we might as well make it more succinct. One less thing to replace I guess if someone proposes another name change.